### PR TITLE
feat: button "New Catch Log Entry" in form (LAN-723)

### PR DESCRIPTION
**Pull request by marination**
### Save and New
With pre-filled organization from last entry

### CODE REVIEW

**Review by scdanieli — DISMISSED** [review]
Suggested changes:

- [x] Add _normal_ save as secondary action
- [x] Fill organization in the newly created doc with the value of the previous doc
- [x] Add translation: Save and New -> Speichern und Neu

File: landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js [hunk 1]
```diff
@@ -2,9 +2,29 @@
...
+			// to avoid primary action refresh on dirty form (setting field value)
+			frm.disable_save();
+
+			frm.page.set_primary_action(__("Save and New"), () => {
+				frappe.run_serially([
+					() => frm.save(),
+					() => frm.reload_doc(),
+					() => frappe.new_doc("Catch Log Entry", {
+						organization: frm.doc.organization
+					}),
```
Comment (scdanieli) [inline landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]:
```suggestion
					}),
					() => frm.refresh(),
```

Without a refresh the buttons for saving in the new form are not shown to me. 🤔

Comment (marination) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: I cleared cache, did a bench build, frappe and erpnext are on version-13-hotfix. Tried switching branch > clearing cache, rebuilding > switching back branch as well. Cannot replicate buttons being absent 😮

Comment (barredterra) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: @marination looks like your bench is too fast and we're facing a race condition on our slow ones:

https://user-images.githubusercontent.com/14891507/217608117-4f6005b3-df54-40ea-9a94-1e3cfc8d160d.mov

Comment (scdanieli) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: Why is @marination having a faster bench? I fell like a second class dev. 😭

Comment (barredterra) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: Probably running on HAL 9000 (https://www.youtube.com/watch?v=njos57IJf-0)

Comment (marination) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: The issue seems to be role based. Frappe tries to render workflow actions as the primary action that's racing with the custom script.🙈

Comment (barredterra) [reply landa/water_body_management/doctype/catch_log_entry/catch_log_entry.js]: Now it's working for admin but not for a "real" user 😄.

I'm afraid we'll have to solve the core issue: frappe loading data and buttons for the old form while we're already on a new one. Let me check with the client if it's worth the effort.

**Review by barredterra — APPROVED** [review]

**Pull request merged**